Cogs/Helpers/websiteHelpers.py

```diff
@@ -33,6 +36,34 @@ async def sendPost(endpoint, json = None):
             return resp.status_code, None
         return resp.status_code, resp.json()["value"]
 ###############################################################################
+async def sendMailchimp(email, status) -> bool:
+    config_object = ConfigParser()
+    config_object.read("BotVariables.ini")
+    variables = config_object["variables"]
```

Comment:
Might be worth refactoring this into a helper since we do it fairly often I believe

Reply:
Nvm just realized this was in the helpers file anyways